applications/serial_lte_modem/doc/SOCKET_AT_commands.rst

```diff
@@ -505,9 +505,34 @@ Syntax
   * ``0`` - Get
   * ``1`` - Set
 
-For a complete list of the supported SET ``<name>`` accepted parameters, see the `SETSOCKETOPT Service Spec Reference`_.
+* The ``<name>`` parameter can accept one of the following values:
 
-For a complete list of the supported GET ``<name>`` accepted parameters, see the `GETSOCKETOPT Service Spec Reference`_.
+  * ``2`` - :c:macro:`SO_REUSEADDR` (set-only).
+    ``<value>`` integer value. ``0`` for disable, ``1`` for enable.
```

Comment:
Since what `<value>` represents changes based on the value of `<name>`, I suggest making it a sub-bullet to make this clearer.
```suggestion
    * ``<value>`` is an integer.
      It is ``0`` when FEATURE is disabled, or ``1`` when it is enable.
```
Something like this for all the definitions. Alternatively:
```suggestion
    * ``<value>`` is an integer that indicates whether FEATURE is enabled.
      It is ``0`` for disable, or ``1`` for enable.
```